docs/debug-guide.md

```diff
@@ -15,8 +15,8 @@ When you see ipfs doing something (using lots of CPU, memory, or otherwise
 being weird), the first thing you want to do is gather all the relevant
 profiling information.
 
-There's a script (`bin/collect-profiles.sh`) that will do this for you and
-bundle the results up into a tarball, ready to be attached to a bug report.
+There's a command (`ipfs diag profile`) that will do this for you and
+bundle the results up into a zip file, ready to be attached to a bug report.
```

Comment:
I assume we're doing zips because either they're more cross platform friendly or the Go tooling is better, right?

Reply:
All of the above:

1. Can be uploaded to slack etc.
2. Can be _streamed_. With a tarfile, I need to know the sizes of files up-front before I can write them. With a zipfile, I can create a file, write it, create the next file, etc. without buffering anything.